pkg/options/options.go

```diff
@@ -0,0 +1,141 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package options
+
+import (
+	"fmt"
+
+	"github.com/spf13/pflag"
+	"github.com/xmudrii/etcdproxy-controller/pkg/controller/etcdproxy"
+	utilerrors "k8s.io/apimachinery/pkg/util/errors"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+// CoreEtcdOptions type is used to wire the core etcd information used by controller to create ReplicaSets.
+type CoreEtcdOptions struct {
+	URL             string
+	CAConfigMapName string
+	CertSecretName  string
+}
+
+// EtcdProxyOptions type is used to pass information from cli to the controller.
+type EtcdProxyOptions struct {
```

Comment:
same question as above: are these the EtcdProxyControllerOptions?

Reply:
Fixeed.